src/css/index.css

```diff
@@ -0,0 +1,67 @@
+* {
+	font-family: Helvetica, sans-serif;
+	box-sizing: border-box;
+	padding: 0;
+}
+
+body {
+	margin: 0;
+}
+
+.navbar {
+	height: 80px;
+	background-color: #6b5b95;
+	display: flex;
+	align-items: flex-end;
+	flex-direction: row;
+}
+
+.search {
+	display: flex;
+	align-items: flex-end;
+	justify-content: flex-end;
+	border-radius: 8px;
+	background-color: #f5f5f5;
+	height: 30px;
+	width: 300px;
+	margin: 5px;
+}
+
+#searchBar {
+	width: 300px;
+	height: 30px;
+	border-top-left-radius: 8px;
+	border-bottom-left-radius: 8px;
+	background-color: #f5f5f5;
+	border: 1px solid grey;
+	border-right: none;
+	outline: 0;
+	font-size: 18px;
+}
+#searchBar:hover,
+#searchBar:focus {
+	border: 1px solid #009688;
+	background-color: white;
+}
+
+.searchButton:hover {
+	border: 1px solid #009688;
+	background-color: white;
+}
+
+div .searchButton {
+	width: 50px;
+	height: 30px;
+	background-color: #f5f5f5;
+	border: 1px solid grey;
+	outline: none;
+	cursor: pointer;
+	padding: 0;
+	border-left: none;
+	border-top-right-radius: 8px;
+	border-bottom-right-radius: 8px;
```

Comment:
Write this as border-radius: 1px 1px 1px 1px as example but change the values to matching size